src/redux/selectors/weatherSelectors.js

```diff
@@ -0,0 +1,14 @@
+import { createSelector } from 'reselect';
+import { getFormattedHourlyData } from '../../utils/hourlyChartSettings';
+
+const gethourlyCityForecast = state => state.weather.hourlyCityForecast;
+
+export const getCityForecast = state => {
+  const { relHumidity, symbol, symbolPhrase, temperature, windSpeed } = state.weather.cityForecast;
+
+  return { relHumidity, symbol, symbolPhrase, temperature, windSpeed };
+};
+
+export const getHourlyChartData = createSelector([gethourlyCityForecast], forecast =>
```

Comment:
OPTIONAL: 
`const getHourlyChartData = createSelector([gethourlyCityForecast], getFormattedHourlyData); `

Reply:
I will remember it for the future, but for now I will leave it for a better understanding